.github/workflows/scan.yml

```
@@ -0,0 +1,48 @@
+name: scan-images
+
+on:
+  schedule:
+    - cron: "0 12 * * 4"
```

```suggestion
    - cron: "0 12 * * 1"
```
Monday is 1. (4 is Thursday).  It would be best to run this before we generally cut the release (Tuesdays).
Ref: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule